service/src/test/resources/uat/output/TC4-9465701610_Powne_cons1only_20210212.xml

```diff
@@ -150,6 +150,11 @@
     <originalText>Happy puppet syndrome</originalText>
 </code>
         
+        <Participant typeCode="PRF" contextControlCode="OP">
+    <agentRef classCode="AGNT">
+        <id root="test-id-3"/>
+    </agentRef>
+</Participant>
```

Comment:
indentation needs to be corrected

Reply:
This is consistent with the rest of the file